src/Panel.tsx

```diff
@@ -85,20 +88,38 @@ const CollapsePanel = React.forwardRef<HTMLDivElement, CollapsePanelProps>((prop
     ...(['header', 'icon'].includes(collapsible) ? {} : collapsibleProps),
   };
 
+  const HeaderWrapper = ({ children: headerWrapperChildren }: PropsWithChildren) => {
```

Comment:
HeaderWrapper not satisfied with our semantic rule. You can just assign the heading role on the `rc-collapse-title` element instead of create new element.